src/main/java/uk/gov/hmcts/darts/audio/service/impl/AudioPreviewServiceImpl.java

```diff
@@ -15,9 +16,15 @@
 public class AudioPreviewServiceImpl implements AudioPreviewService {
 
     private final CachingAudioPreviewEncoder cachingAudioPreviewEncoder;
+    private final MediaIdValidator mediaIdValidator;
 
     @Override
     public AudioPreview getOrCreateAudioPreview(Integer mediaId) {
+        // return 404 if the media is hidden
```

Comment:
Consider removing these comments are they are susceptible to becoming out of sync with the actual behaviour - if the underlying validator logic (or returned HTTP code) is ever changed there's a good chance the dev won't update these comments in step.